### ISSUE TITLE
/warp gives command exception

### ISSUE THREAD

**Issue by hamcoremusic** [Zrips/CMI#4520]
**Description of issue:**
/warp is not working and is generating an error

---

**ERROR (DELETE IF YOU HAVE NO ERROR):**
```
>.... [23:07:16 INFO]: HAMCORE issued server command: /warp
>.... [23:07:16 ERROR]: null
org.bukkit.command.CommandException: Unhandled exception executing command 'cmi' in plugin CMI v8.7.9.4
at org.bukkit.command.PluginCommand.execute(PluginCommand.java:47) ~[patched_1.16.3.jar:git-Paper-251]
at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:159) ~[patched_1.16.3.jar:git-Paper-251]
at org.bukkit.craftbukkit.v1_16_R2.CraftServer.dispatchCommand(CraftServer.java:802) ~[patched_1.16.3.jar:git-Paper-251]
at net.minecraft.server.v1_16_R2.PlayerConnection.handleCommand(PlayerConnection.java:1918) ~[patched_1.16.3.jar:git-Paper-251]
at net.minecraft.server.v1_16_R2.PlayerConnection.a(PlayerConnection.java:1729) ~[patched_1.16.3.jar:git-Paper-251]
at net.minecraft.server.v1_16_R2.PacketPlayInChat.a(PacketPlayInChat.java:47) ~[patched_1.16.3.jar:git-Paper-251]
at net.minecraft.server.v1_16_R2.PacketPlayInChat.a(PacketPlayInChat.java:5) ~[patched_1.16.3.jar:git-Paper-251]
at net.minecraft.server.v1_16_R2.PlayerConnectionUtils.lambda$ensureMainThread$1(PlayerConnectionUtils.java:23) ~[patched_1.16.3.jar:git-Paper-251]
at net.minecraft.server.v1_16_R2.TickTask.run(SourceFile:18) ~[patched_1.16.3.jar:git-Paper-251]
at net.minecraft.server.v1_16_R2.IAsyncTaskHandler.executeTask(IAsyncTaskHandler.java:136) ~[patched_1.16.3.jar:git-Paper-251]
at net.minecraft.server.v1_16_R2.IAsyncTaskHandlerReentrant.executeTask(SourceFile:23) ~[patched_1.16.3.jar:git-Paper-251]
at net.minecraft.server.v1_16_R2.IAsyncTaskHandler.executeNext(IAsyncTaskHandler.java:109) ~[patched_1.16.3.jar:git-Paper-251]
at net.minecraft.server.v1_16_R2.MinecraftServer.ba(MinecraftServer.java:1135) ~[patched_1.16.3.jar:git-Paper-251]
at net.minecraft.server.v1_16_R2.MinecraftServer.executeNext(MinecraftServer.java:1128) ~[patched_1.16.3.jar:git-Paper-251]
at net.minecraft.server.v1_16_R2.IAsyncTaskHandler.awaitTasks(IAsyncTaskHandler.java:119) ~[patched_1.16.3.jar:git-Paper-251]
at net.minecraft.server.v1_16_R2.MinecraftServer.sleepForTick(MinecraftServer.java:1089) ~[patched_1.16.3.jar:git-Paper-251]
at net.minecraft.server.v1_16_R2.MinecraftServer.w(MinecraftServer.java:1003) ~[patched_1.16.3.jar:git-Paper-251]
at net.minecraft.server.v1_16_R2.MinecraftServer.lambda$a$0(MinecraftServer.java:177) ~[patched_1.16.3.jar:git-Paper-251]
at java.lang.Thread.run(Thread.java:834) [?:?]
Caused by: java.lang.NullPointerException
at com.Zrips.CMI.Modules.Warps.WarpManager.getWarp(WarpManager.java:560) ~[?:?]
at com.Zrips.CMI.commands.list.warp.perform(warp.java:85) ~[?:?]
at com.Zrips.CMI.commands.CommandsHandler.onCommand(CommandsHandler.java:340) ~[?:?]
at org.bukkit.command.PluginCommand.execute(PluginCommand.java:45) ~[patched_1.16.3.jar:git-Paper-251]
... 18 more
```
---

**Cmi Version (using`/cmi version`):** 
8.7.9.4
**Server Type (Spigot/Paperspigot/etc):** 
Paper
**Server Version (using `/ver`):** 
Paper(251) 1.16.3-R0.1-SNAPSHOT

**Comment by SmallSansSerif** [comment]
Same error on 8.7.9.4
Tuinity version git-Tuinity-"576e2cc" (MC: 1.16.3) (Implementing API version 1.16.3-R0.1-SNAPSHOT)

https://pastebin.com/3XYLnRXg

**Comment by archroot** [comment]
So I have the same problem :(

https://pastebin.com/x8sca0K9

 CMI: 8.7.9.4 SqLite
 INFO]: Server: Paper(1562) 1.12.2-R0.1-SNAPSHOT

**Comment by AZFire1** [comment]
Yup ^ :)

**Comment by Zrips** [comment]
Will be fixed with 1.16.4 update
